requirements.txt

```diff
@@ -4,19 +4,16 @@
 #
 #    pip-compile
 #
-backports.ssl-match-hostname==3.5.0.1  # via docker-py
-certifi==2019.6.16        # via requests
+certifi==2020.4.5.2       # via docker
 chardet==3.0.4            # via requests
 click==6.6
-docker-py==1.10.6
-docker-pycreds==0.2.1     # via docker-py
-idna==2.8                 # via requests
-ipaddress==1.0.17         # via docker-py
-pyyaml==5.1.1
-requests==2.22.0
+docker==4.2.1
+idna==2.9                 # via requests
+PyYAML==5.3.1
+requests==2.23.0
 retrying==1.3.3
-scrapinghub==2.2.1
```

Comment:
The requirement was pinned to support [multiple jobs cancellation](https://github.com/scrapinghub/shub/commit/46c52389c2846a2fc6d87958ec7fd2aef68ef5b5#diff-b4ef698db8ca845e5845c4618278f29aR18), it won't work with the older versions of the client